Sources/App/Controllers/UserController.swift

```diff
@@ -17,13 +17,10 @@ class UserController: RouteCollection {
         
         
         let authSessionRouter = router.grouped(User.authSessionsMiddleware())
-        authSessionRouter.post("login", use: login)
-        
-        let protectedRouter = authSessionRouter.grouped(RedirectMiddleware<User>(path: "/"))
-        
-        protectedRouter.get("profile", use: profile)
-        protectedRouter.post("updateProfile", use: updateProfile)
         
+        authSessionRouter.post("login", use: login)
+        authSessionRouter.get("profile", use: profile)
+        authSessionRouter.post("updateProfile", use: updateProfile)
```

Comment:
why isn't this protected?

Reply:
API is not handling the UI so the protectedRouter is just `RedirectMiddleware<User>(path: "/")`